app/src/main/java/io/nekohasekai/sagernet/vpn/nav/TermsofServiceActivity.kt

```diff
@@ -0,0 +1,24 @@
+package io.nekohasekai.sagernet.vpn.nav
+
+import android.os.Bundle
+import android.widget.ImageView
+import io.nekohasekai.sagernet.R
+import io.nekohasekai.sagernet.vpn.BaseThemeActivity
+
+class TermsofServiceActivity : BaseThemeActivity() {
```

Comment:
Make it uppser case:
TermsofServiceActivity => TermsOfServiceActivity

Reply:
fixed